pkg/api/app_peers.go

```diff
@@ -14,22 +14,22 @@ type Peer struct {
 	LastSeen uint64 `json:"lastSeen"`
 }
 
-type PeersAll struct {
+type PeersKnown struct {
 	Peers []Peer `json:"peers"`
 }
 
-func (a *App) PeersAll() (*PeersAll, error) {
+func (a *App) PeersKnown() (PeersKnown, error) {
 	peers, err := a.state.Peers()
 	if err != nil {
-		return nil, &InternalError{err}
+		return PeersKnown{}, errors.Wrap(err, "PeersKnown")
```

Comment:
the returned error should be errors.Wrap(err, "failed to get known peers")

Reply:
Better to change error message here